canal/canal_test.go

```diff
@@ -28,6 +28,8 @@ func (s *canalTestSuite) SetUpSuite(c *C) {
 	cfg := NewDefaultConfig()
 	cfg.Addr = fmt.Sprintf("%s:3306", *testHost)
 	cfg.User = "root"
+	cfg.HeartbeatPeriod = 2
```

Comment:
I prefer using 200ms and 300ms instead, so the latter sleep can only 1s, no need 4s.